### PR TITLE
TM-1083 change background color back to grey color, fix tauch scroll …

### PR DESCRIPTION
**Pull request by ArekZc**
…on mobile

### CODE REVIEW

**Comment by marnits** [comment]
You should consider moving `slide-button` into the left in case of simulate green border:
```
#navbar-slide-button {
    ...
    width: $navbar-slide-button-size + 1;
    left: -1px;
    ...
}
```

**Comment by marnits** [comment]
Somebody needs to test scroll on a real device because on Xcode Simulator it doesn't work.

File: src/container-debugger/debug.js [hunk 1]
```diff
@@ -73,7 +73,11 @@
         handleResolution();
 
         wrapper.style.position = 'fixed';
-        //wrapper.style.zIndex = '2147483647';
+
+        wrapper.style['-webkit-overflow-scrolling'] = 'touch';
```
Comment (MarcinMat) [inline src/container-debugger/debug.js]: Are You sure that this line affects style at all? Use desktop safari to inspect the page on emulator.

Anyway, try one of:

```
wrapper.style.webkitOverflowScrolling = 'touch';
```

```
wrapper.style.WebkitOverflowScrolling = 'touch';
```

```
wrapper.setAttribute("style","-webkit-overflow-scrolling:touch")
```

etc.

**Comment by ArekZc** [comment]
If you move to left navbar, you will have space from the right

**Comment by ArekZc** [comment]
@marnits I check it on @emwil iphone and it works

**Pull request merged**